src/fusiondls/settings.py

```diff
@@ -176,3 +176,57 @@ def __post_init__(self):
         Tcool = np.append(0, Tcool)
         Lalpha = np.fromiter((self.cooling_curve(dT) for dT in Tcool), float)
         self.Lz = [Tcool, Lalpha]
+
+
+def make_SparRange(geometry, mode="equally_spaced_poloidal", num_locations=3):
```

Comment:
```suggestion
def make_SparRange(geometry: MagneticGeometry, mode: str = "equally_spaced_poloidal", num_locations: int = 3) -> NDArray[np.floating]:
```

And at the top:

```python
from numpy.typing import NDArray
from .settings import MagneticGeometry
```
... and then resort the imports with `ruff --select=I --fix`

Then we don't need to list types or the `Returns` section in the docstring.